### PR TITLE
Sean/dls 4607

### PR DESCRIPTION
**Pull request by SeanMurray1**

### CODE REVIEW

**Comment by SeanMurray1** [comment]
test this please

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/radio_group_conditional_govuk.scala.html [hunk 1]
```diff
@@ -46,55 +46,56 @@
...
-            @label
-        </legend>
-        @extraContentIdWithContent.map { case (id, text) =>
-        <div id="@id">
-            @text
-        </div>
-        }
-        @hintIdWithHintText.map { case (id, text) =>
-        <span id="@id" class="govuk-hint">
+  <fieldset class="govuk-fieldset" id="@fieldId" @if(describedByKeys.nonEmpty) { aria-describedby="@describedByKeys" }>
```
Comment (adamliptrot-oc) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/radio_group_conditional_govuk.scala.html:52]: Can this use this govukRadios component?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/text_input_govuk.scala.html [hunk 2]
```diff
@@ -42,9 +42,9 @@
 
 @classes = @{if(hasErrors) "govuk-form-group govuk-form-group--error" else "govuk-form-group" }
 @inputClasses = @{if(hasErrors) {
-  classBuilder(Some("form-control"), Some("form-control-error"), inputClass)
+  classBuilder(Some("govuk-input"), Some("govuk-input--error"), inputClass)
 } else {
-  classBuilder(Some("form-control"), inputClass)
+  classBuilder(Some("govuk-input"), inputClass)
```
Comment (adamliptrot-oc) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/text_input_govuk.scala.html:47]: Can this use the govukInput component?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html [hunk 3]
```diff
@@ -93,18 +93,18 @@ <h1 class="govuk-label-wrapper">
...
-              class="govuk-input govuk-input--width-10 @inputClasses"
-              id="@fieldId"
-              name="@fieldId"
-              type="text"
-              inputmode="decimal"
-              @if(describedByKeys.nonEmpty) { aria-describedby="@describedByKeys" }>
+      <div class="govuk-input__prefix @aspect@if(hasErrors){ form-control--error}" aria-hidden="true">
+          @unit
+      </div>
+    <input
```
Comment (adamliptrot-oc) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html:99]: Can this use the govukInput component?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/warning_govuk.scala.html [hunk 4]
```diff
@@ -1,18 +1,30 @@
...
 
-@(warningText: String)(implicit messages: Messages)
-
+@(warningText: String)
 
 @govukWarningText(WarningText(
 iconFallbackText = "Warning",
-content = Text(messages(warningText))
+content = HtmlContent(Html(s"""
+    |<strong class="bold-small" id="warning">
```
Comment (adamliptrot-oc) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/warning_govuk.scala.html:27]: Can this use the govukWarningText component?

Comment (SeanMurray1) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/warning_govuk.scala.html:27]: GovukWarningText is being used here, this is being passed into the content parameter.

Comment (adamliptrot-oc) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/warning_govuk.scala.html:27]: Ah yes sorry missed that :)

**Comment by nubz** [comment]
closing due to lack of activity

**Pull request closed without merging**